source/Nuke.Common/Execution/InjectionAttributeBase.cs

```diff
@@ -22,6 +23,13 @@ namespace Nuke.Common.Execution
     public abstract class InjectionAttributeBase : Attribute
     {
         [CanBeNull]
-        public abstract object GetValue(string memberName, Type memberType);
+        [Obsolete("Use GetValue(string memberName, Type memberType, MemberInfo memberInfo) instead.", error: true)]
```

Comment:
It's probably not called from user code, so won't have any effect. Imo, just remove.